samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +59,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.None();
```

Comment:
Pinpoint called this one Skip/IgnoreAuthentication, c.Ignore() works for me as, seems like a strict improvement over None/Nothing anyways...  => Ignore()/Ignored